src/sentry/static/sentry/app/views/releases/detail/overview/releaseStatsRequest.tsx

```diff
@@ -137,283 +172,252 @@ class ReleaseStatsRequest extends React.Component<Props, State> {
     });
   };
 
-  fetchCountData = async (type: YAxis) => {
-    const {api, yAxis} = this.props;
-
-    const response = await api.requestPromise(this.statsPath, {
-      query: {
-        ...this.baseQueryParams,
-        type,
-      },
-    });
-
-    const transformedData =
-      yAxis === YAxis.SESSION_DURATION
-        ? this.transformSessionDurationData(response.stats)
-        : this.transformCountData(response.stats, yAxis, response.statTotals);
+  async fetchSessions() {
+    const {api, version} = this.props;
 
-    return {...transformedData, crashFreeTimeBreakdown: response.usersBreakdown};
-  };
-
-  fetchRateData = async () => {
-    const {api} = this.props;
-
-    const [userResponse, sessionResponse] = await Promise.all([
-      api.requestPromise(this.statsPath, {
+    const [
+      releaseResponse,
+      otherReleasesResponse,
+    ]: SessionApiResponse[] = await Promise.all([
+      api.requestPromise(this.path, {
         query: {
           ...this.baseQueryParams,
-          type: YAxis.USERS,
+          field: 'sum(session)',
+          groupBy: 'session.status',
         },
       }),
-      api.requestPromise(this.statsPath, {
+      api.requestPromise(this.path, {
         query: {
           ...this.baseQueryParams,
-          type: YAxis.SESSIONS,
+          field: 'sum(session)',
+          groupBy: 'session.status',
+          query: stringifyQueryObject(new QueryResults([`!release:${version}`])),
```

Comment:
Should the version be quoted? Release names can contain spaces which will make queries you don't want.

Reply:
Ah, I thought that stringifyQueryObject+QueryResults is handling that, thanks!